CHANGELOG.rst

```diff
@@ -10,6 +10,12 @@ Development
 - Backend: Migrate from pandas to polars
 - Sources: Add DWD Road Weather data
 
+.. attention::
+
+    Switching to Polars may cause breaking changes for certain user-space code heavily using pandas idioms.
+    For example, ``stations.all().df.loc[:, ["station_id", "latitude", "longitude","height", "name", "icao_id"]]``
+    should be written like ``stations.all().df.select(pl.col("station_id"), pl.col("latitude"), pl.col("longitude"))`` now.
```

Comment:
a) Do you think this admonition makes sense?
b) Isn't there a more compact variant for Polars needing to enumerate all columns/fields like this?
c) Do you have more or other suggestions what to add here?

Reply:
a) I'd believe in the sovereignty of our users meaning we provided the shortest way for people who are more convenient with pandas (``df.to_pandas()``) but we are not supposed to provide a polars -> pandas tutorial. ``pl.select`` is a really basic feature and people should be able to look it up from the polars docs.

b) there would be ``df.select(["colA", "colB"])`` as alternative without the option to modify the columns themselves.

c) Maybe a warning sign that instead of a pandas DataFrame wetterdienst now returns a polars DataFrame by default? This could be placed in the README head.

Reply:
Thank you. I rephrased the message, and also added it to the README.